src/utils/camelcaseObject.ts

```diff
@@ -1,5 +1,3 @@
 import humps from "lodash-humps-ts";
 
-export const camelcaseObject = (object: object) => {
-  return humps(object);
-};
+export const camelcaseObject = (object: object) => humps(object);
```

Comment:
I think we shouldn't test a library, this is also a fault of mine because we don't really need to create a function if it's just going to pass to another function. Also, i think i should only call the humps function once in the `axios.ts` file rather than calling it multiple times in the `features/*` so everything gets camelCased immediately. What do you think?

Reply:
Yes, I think it should be called in the axios response only one time. (for now, we just have a couple of calls but it can increase).
And I think it's a better idea as it is having the humps function in our implementation because if in the future for some reason we want to change the humps library we will change it in one single place instead of searching it across the code.

Reply:
Good argument! Alright, i will just open up an issue regarding refactoring the axios & humps function so those functions are not everywhere.